libs/ui-watt/src/lib/components/dropdown/watt-dropdown.component.ts

```diff
@@ -132,15 +139,21 @@ export class WattDropdownComponent
    * @ignore
    */
   ngOnInit(): void {
-    // load the initial list of options
-    this.filteredOptions$.next(this.options.slice());
-
     this.listenForFilterFieldValueChanges();
     this.initializePropertiesFromParent();
     this.bindParentValidatorsToControl();
     this.bindControlToParent();
   }
 
+  /**
+   * @ignore
+   */
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes.options.currentValue !== changes.options.previousValue) {
```

Comment:
issue: this will result in an error if a property other than `options` is updated dynamically. Make access to `currentValue` and `previousValue` optional.

```suggestion
    if (changes.options?.currentValue !== changes.options?.previousValue) {
```